doc/design/model_format.md

```diff
@@ -33,4 +33,6 @@ In detail, tensor's  byte view as the table shows. Note that all the signed valu
 
 ## Summary
 
-We introduce the model format, the `ProgramDesc` describe the **topology**, and a bunch of particular format binary tensors describes the **parameters**.
+- We introduce a model format.
+- The `ProgramDesc` describe the model **topology**. 
```

Comment:
I think `ProgramDesc` is not for describing the model, but the computation procedure.  Here we are using the fact that a DL model can be represented by the forward-pass, so could we represent the model by a `ProgramDesc` containing the forward-pass.  So we might want to replace

> The `ProgramDesc` describe the model **topology**

into 

> The model represented by its forward-pass computation procedure saved in a `ProgramDesc` protobuf message.